README.md

```diff
@@ -133,6 +133,16 @@ Debug the e2e operator tests in [Goland](https://www.jetbrains.com/go/)
 | `make test/coverage/prepare` | Prepares coverage report from unit and e2e test results     |
 | `make test/coverage`         | Generates coverage report                                   |
 
+##### Running tests without cluster admin permissions
+It's possible to deploy CRDs, roles, role bindings, etc. separately from running the tests:
+1. Run `make cluster/prepare` as a cluster admin.
+2. Run `make test/e2e-image-no-admin` as a user. The user needs the following permissions to run te tests:
+```
+apiGroups: ["", "apps", "keycloak.org"]
+resources: ["persistentvolumeclaims", "deployments", "statefulsets", "keycloaks", "keycloakrealms", "keycloakusers", "keycloakclients", "keycloakbackups"]
```

Comment:
Could you please tell me how this is used? Perhaps we don't need to mention it in the docs? Or maybe, it's better to write a `role.yaml` file dedicated to this use case?

Reply:
This is just not forget what permissions are required to run the tests without cluster admin permissions. I'd say let's leave it to the cluster admins how will they use this – whether they create a new role or add a new rule to an existing role.

Reply:
Ok, understood. Thanks!